### DISCUSSION TITLE
Say thanks to CP Editor!

### DISCUSSION THREAD

**Discussion by ouuan** [cpeditor/cpeditor#755] [Show and tell]
As discussed in #637, this is the correct place to say thanks to developers.

Feel free to express your story and gratitude here! :rocket:

**Comment by Mr-Bangladesh** [comment]
just wow!!

**Reply by Marhoosh** [reply]
3Q

**Comment by bishojet** [comment]
amazing

**Comment by Al-Sharif-Muhammed** [comment]
amazing <3 <3

**Comment by abhiiguptaa** [comment]
I like your editor very and I also wanted to use your editor but I can't use it because it does not support the vim motions or emulations like h, j, k, l binding to move the cursor up and down and other useful key. Everything is perfect except this. Please add this feature in the upcoming updates because I wanted to use the editor which is light weight and easy to use. Thanks

**Comment by Yash-bhagwat** [comment]
Thanks, much love!

**Comment by Muaath5** [comment]
Thank you so much.
A powerful tool that I was searching these days

**Comment by DevGarg7** [comment]
thanks

**Comment by aqua4** [comment]
Thanks!!!

**Comment by khushal1512** [comment]
thanks <3

**Comment by 0ssm** [comment]
Just wanted to say a big thank you to the amazing developers of CP Editor. It's fast, clean, and makes competitive programming so much easier. You’ve built something truly special for the community.

Grateful for all your hard work
